lib/administrate/version.rb

```diff
@@ -1,3 +1,3 @@
 module Administrate
-  VERSION = "0.2.0".freeze
+  VERSION = "0.2.1".freeze
```

Comment:
Yeah. i didn't really want to jump too far until we had some of @graysonwright's guidance... however i have definitely noticed a trend in thoughtbot repos to sit with PRs for, well, a long time.